From 01cd6ca28a692352ef1be550d33ca2a8ec95d5e0 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 10 Jun 2024 17:41:14 +0100
Subject: [PATCH] drivers: media: pcie: hailo: Fix include paths

An attempt to fix the include paths - they look reasonable, but the
GitHub auto-builds fail.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/media/pci/hailo/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/media/pci/hailo/Makefile
+++ b/drivers/media/pci/hailo/Makefile
@@ -25,8 +25,8 @@ hailo_pci-objs += $(VDMA_SRC_DIRECTORY)/
 
 ccflags-y      += -Werror
 ccflags-y      += -DHAILO_RASBERRY_PIE
-ccflags-y      += -I$(srctree)/$(src)
-ccflags-y      += -I$(srctree)/$(src)/include
-ccflags-y      += -I$(srctree)/$(src)/common
+ccflags-y      += -I $(src)
+ccflags-y      += -I $(src)/include
+ccflags-y      += -I $(src)/common
 
 clean-files := $(hailo_pci-objs)
